include/LDtkLoader/DataTypes.hpp

```diff
@@ -102,6 +103,8 @@ namespace ldtk {
         auto extension() const -> std::string;
     };
 
+    typedef std::function<auto(const std::string&) -> std::string> FileLoader;
```

Comment:
I vehemently agree. I only used `typedef` because there exist a few of them in the `DataTypes.hpp` file already and I tried to keep consistency. Would you like me to change all the other `typedef`s to `using` declarations as well?

Reply:
so it seems my past self used typedef instead of using, and you were right to keep the consistency, my bad.
Let's keep it a typedef, changing the typedefs in this file to using can be made in another PR.